### DISCUSSION TITLE
stuck at current httpx version v1.3.5, file to big?

### DISCUSSION THREAD

**Discussion by r4w7ed** [projectdiscovery/httpx#1394] [Q&A]
Hi, i'm trying to scan whole ip from 10.1.1.1/24 to 10.1.255.1/24 using httpx, but stuck at "Current httpx version v1.3.5"
i think it beacuse the list is too big, any tips?

my command: ./httpx -l local.txt -fr -title -o local-found
inside local.txt:
10.1.1.1/24
10.1.2.1/24
10.1.2.1/24
...
10.1.254.1/24
10.1.255.1/24

**Comment by MetzinAround** [comment]
What happens when you run it in verbose mode? It should give you more insight into what's happening /where it's hanging. 
`https -l local.txt -fr -title -o local-found -v`

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
